### ISSUE TITLE
Spaces | It might make sense for the notification tab to be space independent

### ISSUE THREAD

**Issue by SimonBrandner** [vector-im/element-android#3204]
Currently, the notifications tab filters by space. But I feel like I would expect notifications to be global.

**Comment by dkasak** [comment]
I don't think this is a good idea since it would result in a loss of functionality. I often use spaces as a way to context filter -- for instance, selecting a Personal space in order to hide all work related stuff. I wouldn't want to see work-related notifications in this view.

On the other hand, you can currently easily get a view of all your notifications by selecting the Home "space".
